### PR TITLE
Butterfly space (S143) is not simply connected (P200)

### PR DESCRIPTION
**Pull request by GeoffreySangston**
Some things need to be edited still. It would be nice instead of handwaving that the circular paths are continuous, to handwave that the quarter-circles have standard continuous deformation retractions onto the endpoints at $y = R$, which assemble into a deformation retraction onto the line $y = R$. 

I really don't know how to handwave this any better than that though, so suggestions are welcome. I'm unlikely to ask this on MSE since it frequently ends up being an unpleasant experience there for a dozen reasons and because I feel like I already know the argument. If that's what we want to do though, we should just close this PR and somebody else can go ahead.

And the arc connected trait can be deleted since the space is Hausdorff.

### CODE REVIEW

**Comment by prabau** [comment]
Tell me is the following idea would work.

First do a deformation retraction of $X$ onto the $x$-axis along vertical lines.  (using a deformation retraction of $[0,+\infty)$ onto $\\{0\\}$ on each vertical line.  I imagine that's easy to see.  Some handwaving would be ok).
Then the $x$-axis with the subspace topology from $X$ has the Euclidean topology of $\mathbb R$, and so again deformation retracts to a point.

**Comment by prabau** [comment]
Hmm, I think my suggestion would not work at all, because each point on the real line is isolated in its vertical line.

**Comment by prabau** [comment]
I can ask a mathse question later today, see what people come up with.  Need to be away from my desk now.

**Comment by danflapjax** [comment]
Note that circular paths away from a point on the x-axis aren't continuous for all radii, as it needs to (locally) lie below the curve $x^2+y^2=(x/y)^2$. See this graph: https://www.desmos.com/calculator/gdkud6jous

**Comment by prabau** [comment]
https://math.stackexchange.com/questions/4998216/is-the-butterfly-space-contractible
Feel free to add an answer there and let's see what other people come up with also.

**Comment by prabau** [comment]
> Note that circular paths away from a point on the x-axis aren't continuous for all radii, as it needs to (locally) lie below the curve x 2 + y 2 = ( x / y ) 2 . See this graph: https://www.desmos.com/calculator/gdkud6jous

Nice desmos graphs.  Some of the green circular paths are stay locally below the blue curve, right?

**Comment by prabau** [comment]
https://math.stackexchange.com/questions/4998216/is-the-butterfly-space-contractible
MartianInvader has finally given a nice correct proof that the butterfly space is not contractible (and not simply connected).

Please upvote his answer if you can.

**Comment by Almanzoris** [comment]
Really interesting! Your question has also become really popular, at least, to my judgement.

**Comment by danflapjax** [comment]
> MartianInvader has finally given a nice correct proof

This feels a little backhanded toward a certain other user...

**Comment by prabau** [comment]
Apologies @danflapjax. Your proof was also a very useful first step, not fully complete but a crucial step in pointing to the right direction.

**Comment by GeoffreySangston** [comment]
I'm sorry everyone I'm just very busy with other obligations these days. I'll try to contribute to this later tonight.

**Comment by prabau** [comment]
No problem at all.

**Comment by GeoffreySangston** [comment]
What a turn of events. It looks like David Gao's second-to-last comment has the same idea I had, but the deformation retractions you form this way aren't continuous. Is the lack of continuity similar to your second comment (though different because in your comment you're talking about a retraction onto the $x$-axis)?

**Comment by prabau** [comment]
David Gao proposed multiple solutions in comments, then deleted some of them as they turned out to not work.  I think the first one was retracting onto the $x$-axis.  Lots of people thought of retractions onto one axis or the other.

**Comment by prabau** [comment]
Can you change the title of the PR?

**Comment by prabau** [comment]
If you move it out of Draft, I'll do the final review.

**Comment by GeoffreySangston** [comment]
I think others are far more qualified than me now to add any other interesting information to the page.

**Review by prabau — APPROVED** [review]

**Comment by prabau** [comment]
It was a stumper!

**Comment by GeoffreySangston** [comment]
This likely common query is getting increasingly interesting

[π-Base, Search for `path connected + ~Simply connected`](https://topology.pi-base.org/spaces?q=path+connected+%2B+%7ESimply+connected)
